### PR TITLE
Add coding style

### PR DESCRIPTION
**Pull request by agavignet**
Development sponsored by https://github.com/Genymobile as part of the #hacktoberfest.

Use black to reformat existing codebase

### CODE REVIEW

File: ci/blackify [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -ev
+cd $(dirname $0)/..
+
+black -l 120 .
```
Comment (CedricCabessa) [inline ci/blackify]: I prefer, the default (80)

I know it is an endless debate, but this what pep8 says and it is the default for Black as well, so better use the default and avoid the `-l`. This avoid discussion :smile:

File: requirements/dev.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+black==19.3b0; python_version >= '3.6'
```
Comment (CedricCabessa) [inline requirements/dev.txt]: better use the last version no? (no pinning)
It's not like it will break something

File: requirements/dev.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (CedricCabessa) [inline requirements/dev.txt]: useless blank

**Comment by CedricCabessa** [comment]
Thank you for your contribution dear stranger :rofl: 

Do you use it in your company? I'd love to ear your story :-)

Tell the other I say hi with a virtual :coffee:

File: requirements/dev.txt (hunk 2 above)
Comment (agavignet) [reply requirements/dev.txt]: assuming different version will not format different way, yes it will not break

**Comment by agavignet** [comment]
> Thank you for your contribution dear stranger

:hugs: 

> Do you use it in your company? I'd love to ear your story :-)

Not yet, one day may be :) we are evaluating your cool project ;)

> Tell the other I say hi with a virtual coffee

:heavy_check_mark:

**Pull request merged**